src/components/WhatWeHelp/whatWeHelp.scss

```diff
@@ -92,6 +93,7 @@
     padding: 1.5rem 1.3rem;
     margin-top: 3rem;
     margin-left: 2rem;
+    border: 0.75px solid #fff;
```

Comment:
zamiast określać kolor na biały, lepiej byłoby dać wartość transparent. Dzięki temu jak zmienisz kolor tła strony, to wszystko zadziała.

Pro Tip: Widziałem, że masz takie ustawienie w kilku miejscach kodu.